transform/models/staging/building_footprints/_building_footprints__models.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+
+sources:
+  - name: building_footprints
+    database: raw_dev
```

Comment:
Thanks for that example! I will skip this since based on our other discussion we don't need a staging table for footprints data and can directly reference it.

Reply:
We still need this: whether or not we have a `stg` model, we still need to document our sources in order to refer to them with the `source` macro